test/zone-test.ts

```diff
@@ -203,7 +203,9 @@ describe('napajs/zone', function () {
             return napaZone1.execute('./napa-zone/test', "broadcastTransportable", ['node']);
         });
 
-        it('@node: -> node zone with anonymous function having closure (should fail)', () => {
+        // This will not fail any more because current implementation uses function `eval()`.
```

Comment:
As explained in comments, we may use `Function()` instead of `eval()` in future. We can re-enable this test then.

Actually there is inconsistency of the context management in current Napa.js for `broadcast()` and `execute()`. The different is, broadcast() uses the global context and execute() uses a local context. This means `broadcast("var a = 0;");` will put variable a in the global context.

I want to fix this eventually, but not in this change. It will be a breaking change since all current broadcast usage may be broken. Scripts need to be changed from
```js
zone.broadcast("var f = function(){ ... };");
zone.broadcast("var x = 123;");
zone.broadcast("var m = require(' ... ');");

zone.execute(()=>{
    global.f();
    var y = global.x;
    global.m.do_something();
});
```
to
```js
zone.broadcast("global.f = function(){ ... };");
zone.broadcast("global.x = 123;");
zone.broadcast("global.m = require(' ... ');");

zone.execute(()=>{
    global.f();
    var y = global.x;
    global.m.do_something();
});
```